iOS Client/TabController/MBTabBarController.swift

```diff
@@ -21,24 +21,19 @@ class MBTabBarController: UITabBarController, StoreSubscriber {
     }
     
     override func viewWillAppear(_ animated: Bool) {
-        MBStore.sharedStore.subscribe(self) {subscription in subscription.select {state in state}}
     }
     
     override func viewWillDisappear(_ animated: Bool) {
-        MBStore.sharedStore.unsubscribe(self)
+
     }
 
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
     
-    // MARK: - StoreSubscriber
-    
-    func newState(state: MBAppState) {
-        if self.selectedIndex != state.navigationState.tabIndex {
-            self.selectedIndex = state.navigationState.tabIndex
-        }
+    func select(tab: Tab) {
+        self.selectedIndex = tab.index
```

Comment:
investigate going `Tab: Int` and using `rawValue`. I might be missing something about why we would need String to be the raw value type . . .

Reply:
I like this better, good call